libs/shared/src/entities/messages/messages.interface.ts

```diff
@@ -33,6 +32,7 @@ export interface IMessageCTA {
   type: ChannelCTATypeEnum;
   data: {
     url?: string;
+    target?: UrlTarget;
   };
```

Comment:
This should use the `Redirect` typing

Reply:
In the Redirect we have the url as required, but in the message entity it is not required because it is the old V1 cta buttons.

Reply:
We will use the Redirect type once we introduce the new fields that we discussed yesterday about aligning the DSL